README.md

```diff
@@ -39,6 +39,7 @@ Add your typekit kitId to `config/environment.js` and you're good to go. A coupl
 
 * `enabled` (Default: `true`): Enable mixpanel tracking
 * `autoPageviewTracking` (Default: `true`): Enable automatic pageview tracking
+* `pageViewAttribute` (Default: `url`): Use some other attribute instead of URL for pageViewTracking
```

Comment:
Also: what object is `this` (the context off which `url` hangs)? `Ember.Router`? `Ember.Route`?